### ISSUE TITLE
Creating a blank team causes issue

### ISSUE THREAD

**Issue by mlee34** [JuiceMind/JuiceMind#66]
The following occurs: 

![Image](https://github.com/JuiceMind/JuiceMind/assets/170837121/55c26dd8-acc8-4dc1-8161-dddd3777c01c)

**Comment by juicemind-dev** [comment]
Yes, our system is looking for an item that doesn't exist. It needs to revert to the "Create new item" screen automatically. Bug will be fixed shortly and updated here.

Thanks!
